### ISSUE TITLE
Failed to build under python3

### ISSUE THREAD

**Issue by jllorente** [ldx/python-ebtables#3]
Hi,

I'm trying to build this under my virtual environment with python3 but it fails.
Your other library python-iptables works well tho :) 

```
$ python3 setup.py build
Traceback (most recent call last):
  File "setup.py", line 7, in <module>
    import ebtables
  File "/home/ubuntu/workspace/python-ebtables/ebtables.py", line 88, in <module>
    _hash = hashlib.sha1('\0'.join([_cdef, _verify])).hexdigest()
TypeError: Unicode-objects must be encoded before hashing
```

**Comment by caniculari** [comment]
I have modify that line in ebtables.py to...

_hash = hashlib.sha1(('\0'.join([_cdef, _verify])).encode('utf-8')).hexdigest()

But still facing so much errors, in the last version. So far I'll try to work with 0.2.0 and try to debug this version if possible.

**Comment by fariello** [comment]
I was able to fix this issue by changing **line 88** from:
`_hash = hashlib.sha1(('\0'.join([_cdef, _verify]))).hexdigest()`
to
`_hash = hashlib.sha1(_cdef.encode('utf-8') + _verify.encode('utf-8')).hexdigest()`
but 
`_hash = hashlib.sha1(('\0'.join([_cdef, _verify])).encode('utf-8')).hexdigest()`
is a better solution. After that there is only one additional error in 0.2.0 for python3. Change **line 102** from:
`_ebtc.ebt_errormsg[0] = '\0'`
to
`_ebtc.ebt_errormsg[0] = '\0'.encode("ascii")`
and it should install for you. If these changes can be incorporated into 0.2.1, this will allow ebtables to be python3 installed.

**Comment by ldx** [comment]
Fixed in #5
